### DISCUSSION TITLE
Dockerized version

### DISCUSSION THREAD

**Discussion by ServerGeek82** [GlitterWare/Passy#94] [Ideas]
Would it be possible as a avid self-hoster to containerize the project so it could run in server mode and sync across devices? Did not see this idea posed yet.

**Comment by GleammerRay** [comment]
I believe that this will most probably be possible. I might try it out myself as well later this week.

**Reply by ServerGeek82** [reply]
That would be awesome running in server mode and devices sync to it would make this amazing.

**Reply by cvx35isl** [reply]
any sketches on this?

i'd be willing to try that myself. Dockerfiles and K8s manifests I can develop.
